pkg/printer/printer.go

```diff
@@ -168,10 +166,28 @@ func GetFlowType(f v1.Flow) string {
 		}
 		return l7Protocol + "-" + l7Type
 	}
-	if f.GetVerdict() == pb.Verdict_DROPPED {
-		return api.DropReason(uint8(f.GetEventType().SubType))
+
+	switch f.GetEventType().GetType() {
+	case api.MessageTypeTrace:
+		return api.TraceObservationPoint(uint8(f.GetEventType().GetSubType()))
+	case api.MessageTypeDrop:
+		return api.DropReason(uint8(f.GetEventType().GetSubType()))
+	case api.MessageTypePolicyVerdict:
```

Comment:
would be nice if we can somehow reuse the logic from https://github.com/cilium/cilium/blob/e831859b5cc336c6d964a6d35bbd34d1840e21b9/pkg/monitor/datapath_policy.go#L104

Reply:
Yes, unfortunately that function is not exported. I can open a PR on Cilium to make it public and undo the logic, but I don't think this should block this PR.

Reply:
no not a blocker